src/commands/no-shorthand.md

```diff
@@ -0,0 +1,22 @@
+# `no-shorthand`
+
+Expand object shorthand properties to their full form.
+
+## Triggers
+
+- `/// no-shorthand`
+- `/// ns`
```

Comment:
```suggestion
- `/// nsh`
```

Maybe this should be more explicit. I am afraid that `ns` could mean many different stuff in the future

Reply:
Changed to `nsh`